rdrf/rdrf/views/registry_view.py

```diff
@@ -14,7 +15,7 @@ def get(self, request, registry_code):
         registry_model = get_object_or_404(Registry, code=registry_code)
 
         context = {
-            'splash_screen': registry_model.splash_screen,
+            'splash_screen': Template(registry_model.splash_screen).render(Context({'home_page_link': request.user.default_page})),
```

Comment:
I don't think this is necessary. You can just define a link in the splash page with: `href="/router/"` and it should do the same thing

Reply:
I think I'm wrong here. It would just redirect to the splash page for a patient/carer.

Reply:
Yep. Alternative might be to use a template tag for the home page link, or render and just use `{{request.user.default_page}}` in the template.

Reply:
I like the idea of just rendering and using `{{request.user.default_page}}` as it's already available.